images/base/Dockerfile.centos7

```diff
@@ -4,9 +4,9 @@
 #
```

Comment:
Agreed, we should be able to remove `.centos7` and update the normal ones to match.